apps/verticalhome/test/marionette/lib/verticalhome.js

```diff
@@ -150,11 +149,6 @@ VerticalHome.prototype = {
     });
   },
 
-  focusRocketBar: function() {
```

Comment:
please, when removing something, double check that no 'tails' remaining at least in gaia folder.
this particular function invoked from 10 files in gaia/apps/search/test/marionette folder & in gaia/apps/system/test/marionette/lib/rocketbar.js